yarn.lock

```diff
@@ -41,7 +41,7 @@
     webpack-filter-warnings-plugin "1.2.1"
     webpack-rtl-plugin "1.8.0"
 
-"@automattic/calypso-color-schemes@^1.0.0":
```

Comment:
I don't really understand what triggered that change?

Reply:
I added the direct dependency.

This lines says that both required versions, (with `^` from a package and without as the direct dependency) resolve to the following lines.